kotlin/internal/toolchains.bzl

```diff
@@ -195,3 +195,26 @@ def define_kt_toolchain(
         toolchain = impl_name,
         visibility = ["//visibility:public"],
     )
+
+def kt_configure_toolchains():
+    """
+    Defines the toolchain_type and default toolchain for kotlin compilation.
+    """
+    native.config_setting(
+        name = "builder_debug_timings",
+        values = {"define": "kt_timings=1"},
+    )
+
+    native.config_setting(
+        name = "builder_debug_trace",
+        values = {"define": "kt_trace=1"},
+    )
+
+    native.toolchain_type(
```

Comment:
It's not moved -- the invocation is still in the same place, but the code isn't repeated.

Once we sort out the layout this will be resolved. Till then, it makes more sense to keep a single source of truth.

Reply:
Make's sense.